ivy_tests/test_ivy/test_frontends/test_numpy/test_mathematical_functions/test_sums_products_differences.py

```diff
@@ -329,3 +329,43 @@ def test_numpy_diff(
         x=x[0],
         axis=axis,
     )
+    
+    
+# ediff1d
+@handle_frontend_test(
+    fn_tree="numpy.ediff1d",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"), 
+        min_num_dims=1, 
+        max_num_dims=1
+    ),
+    to_end=helpers.ints(
+        min_value=-1,
+        max_value=10,
+    ),
+    to_begin=helpers.ints(
```

Comment:
To be totally correct, `to_end` and `to_begin` should be either an integer (which you've done) or a sequence. You could use `st.one_of` like so `st.one_of(st.integers(-1, 10), st.lists(st.integers(-1, 10), min_size=1, max_size=10))`.